### PR TITLE
Define pc.getTransceivers() et al to be in insertion order.

### PR DESCRIPTION
**Pull request by jan-ivar**
Fix for https://github.com/w3c/webrtc-pc/issues/1669.

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -4847,13 +4837,10 @@ <h2>Methods</h2>
...
-                represents a snapshot of all the
-                <code><a>RTCRtpTransceiver</a></code> objects in this <code><a>
-                  RTCPeerConnection</a></code> object's <a>set of
-                  transceivers</a>. The conversion from the transceiver set to
-                  the sequence is user agent defined and the order does not
-                  have to be stable between calls.
+                <li>Let <var>transceivers</var> be a new sequence consisting of
+                all <code><a>RTCRtpTransceiver</a></code> objects in this
+                <code><a>RTCPeerConnection</a></code> object's
+                <a>set of transceivers</a>, in insertion order.
```
Comment (stefhak) [inline webrtc.html]: Is `insertion order` defined somewhere?

**Comment by alvestrand** [comment]
Ready to merge. (Github is acting up)

File: webrtc.html (hunk 1 above)
Comment (jan-ivar) [reply webrtc.html]: It's the order our algorithms add transceivers to the *set of transceivers*. This seems well-defined in our spec for local "add*" methods, and in JSEP for ones added by remote media descriptions.

Comment (stefhak) [reply webrtc.html]: works for me!

**Comment by stefhak** [comment]
Just noting that the order of transceivers may be different between the two endpoints, and may differ from m-line order. I guess that's ok.

**Pull request merged**